packages/components/tsconfig.eslint.json

```diff
@@ -1,4 +1,4 @@
 {
   "extends": "./tsconfig.json",
-  "include": ["src", "test", "**/*.js", "**/*.ts"]
+  "include": ["src", "test", "**/*.js", "**/*.ts", "**/*.tsx"],
```

Comment:
Technically not legal json to have trailing comma

Reply:
fixed